tools/ldb_cmd.cc

```diff
@@ -3519,11 +3519,16 @@ DBFileDumperCommand::DBFileDumperCommand(
     const std::vector<std::string>& /*params*/,
     const std::map<std::string, std::string>& options,
     const std::vector<std::string>& flags)
-    : LDBCommand(options, flags, true, BuildCmdLineOptions({})) {}
+    : LDBCommand(options, flags, true,
+                 BuildCmdLineOptions({ARG_DECODE_BLOB_INDEX})),
+      decode_blob_index_(false) {
```

Comment:
We could simply do `decode_blob_index_(IsFlagPresent(flags, ARG_DECODE_BLOB_INDEX))` here (note that `IsFlagPresent` is a static helper)

Reply:
Good point, thank you for the suggestion!